pytrip/tripexecuter/execparser.py

```diff
@@ -243,7 +245,9 @@ def _parse_field(self, line):
         if len(items) > 1:
             if "new" in items[1]:
                 logger.debug("Parse a new field number {:d}".format(_number))
-                field = Field()
+                projectile = Projectile("C")
```

Comment:
Use protons (without RiFi) as a default kernel

Reply:
Because you cannot create empty projectile

Reply:
I have got an idea how to resolve this

Reply:
I think that the problem here is related to a more general issue - whether the object of class should be fully initialised after a call to `__init__`.

The best approach is to first set all atributes using default (reasonable) values and later update them to a custom ones. Setting kernel to a default (not-None) value is one of possibilities, but I'd do it without specifying explicitely what kind of projectile it is. I'd rather use following code:

```
projectile = Projectile()
kernel = KernelModel(projectile)
field = Field(kernel)
```

or even better:
```
field=Field()
```
This require implementation of respective `__init__` methods which when no argument is provided, would use some reasonble defaults (i.e. 'H' as a projectile name).


Take a look at `_field_args` https://github.com/pytrip/pytrip/blob/master/pytrip/tripexecuter/execparser.py#L92 and `_parse_extra_args` and `_update_obj` methods. 

Parsing of projectile argument would now add an attribute called `projectile` to the `self.plan.fields[-1]` object (if I am correct). 

What do you think of using a different method that a `_update_obj` for a "projectile" key in `_field_args` ?

Such method (i.e. called `_update_kernel_obj`) could look like:
```
def _update_kernel_obj(_obj, _objattr, _val, _format):
    _update_obj(_obj.kernel, _objattr, _val, _format)
```

Reply:
Another idea is to keep an `projectile` attribute of `Field` and add apriopriate `@property` and `@..setter` decorators which will make changes and read data from `kernel`

Reply:
A bit of history:
Projectile was specied on plan level intially, since TRiP did not have multi-particle optimization. Therefore it was kept on plan level for simplicity. 
However, this has changed for some TRiP versions to my knowledge.

It would indeed be better to leave this completely and specify it on field level only, as this follows TRiP logic.

Notice, this will also make the GUI change a bit, since so far it was only specified on plan level.

A solution could be to add a method/setter on plan level which would overwrite the field values.

Reply:
Introduced default constructors. I will update pytripgui after merging this PR